CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Unreleased
+* tba
+
+# 0.22.0
+
+* support for metadata info on sarif result
+* Makefile fix for go mod tidy
+* test without Kind by default
+* deprecate kubernetes.io in override labels
+
+
+# 0.21.0
```

Comment:
this is not being used to automatically populate the draft release. However, it'd be nice to keep as a way to document things.  Should we keep it or not?